HymnsTests/Display/DisplayHymnViewModelSpec.swift

```diff
@@ -135,6 +155,49 @@ class DisplayHymnViewModelSpec: QuickSpec {
                                     it("should observe its favorite status") {
                                         verify(favoritesStore.observeFavoriteStatus(hymnIdentifier: newSong145, action: any())).wasCalled(exactly(1))
                                     }
+                                    it("Piano url should be http://www.hymnal.net/en/hymn/c/145/f=ppdf") {
+                                        expect(target.chordsUrl).to(equal(URL(string: "http://www.hymnal.net/en/hymn/c/1151/f=gtpdf")))
+                                    }
+                                    it("Guitar url should be http://www.hymnal.net/en/hymn/c/145/f=pdf") {
+                                        expect(target.guitarUrl).to(equal(URL(string: "http://www.hymnal.net/en/hymn/c/1151/f=pdf")))
+                                    }
+                                    it("Text url should be http://www.hymnal.net/en/hymn/c/145/f=gtpdf") {
+                                        expect(target.pianoUrl).to(equal(URL(string: "http://www.hymnal.net/en/hymn/c/1151/f=ppdf")))
+                                    }
```

Comment:
FYI, I generally like making the expected url a constant and then just pass it in to the assertions. That way, you can avoid errors like this, where you say "**text** url should be ...c/**145**/..." but you're asserting on **pianoUrl** being "...c/**1151**/..."